templates/tailwind.config.js.ejs

```diff
@@ -0,0 +1,42 @@
+// AUTO-GENERATED - File generated by figma2theme <%- version %>, manual edits may be overwritten
```

Comment:
couldn't figure out the prettier config to make it stop autoformatting these ejs files so wierdly.

Reply:
You can add a `.prettierignore` file to the root with`*.ejs` as the contents to fix this 👍

Reply:
There already is one with ejs in there